setup.py

```diff
@@ -35,6 +35,11 @@
     long_description_content_type="text/markdown",
     url="https://github.com/deekshithanand/flask-create.git",
     packages=find_packages(),
+    package_data={
+        "flask_create": [
```

Comment:
Uhmm, this is obsolete for now. All package data are added in MANIFEST file. So it is not necessary here!
More info [here!](https://docs.python.org/2/distutils/setupscript.html).

Reply:
I added this because the wheel did not include the template files. Fixed the manifest and removed `package_data` parameter.